tests/integration/records/test_views.py

```diff
@@ -1123,3 +1125,197 @@ def test_citation_summary_facet_filters(api_client, db, create_record_factory):
     response_data_citation_summary = response_data["aggregations"]["citation_summary"]
     assert response_status_code == 200
     assert response_data_citation_summary == expected_citation_summary_aggregation
+
+
+def test_create_literature_record_through_api(api_client, db, create_user_and_token):
+    data = faker.record("lit")
+    token = create_user_and_token()
+    headers = {"Authorization": f"Bearer {token.access_token}"}
+    content_type = "application/json"
+    response = api_client.post(
+        "literature", json=data, headers=headers, content_type=content_type
+    )
+    expected_data = deepcopy(data)
+    expected_data["control_number"] = response.json["id"]
+    del expected_data["_collections"]
+
+    assert response.status_code == 201
+    assert response.json["metadata"] == expected_data
+
```

Comment:
For completeness, I would also check that pids are generated and the record is properly indexed, WDYT?

Reply:
Done in the integration-async tests :taco: